ntbs-integration-tests/ClinicalDetailsPageTests.cs

```diff
@@ -177,31 +177,44 @@ public async Task Post_ClearsConditionalInputValues_IfRadiosSetToOtherValue()
             Assert.Equal("", ((IHtmlInputElement)reloadedDocument.GetElementById("FormattedDeathDate_Year")).Value);
         }
 
-        public static IEnumerable<object[]> InvalidDates()
+        [Fact]
+        public async Task IfInvalidDate_ValidateClinicalDetailsDate_ReturnsValidDateErrorMessage()
         {
-            yield return new object[] { "1", "0", "2009", ValidationMessages.ValidDate };
-            yield return new object[] { "1", "1", "2009", ValidationMessages.DateValidityRange(ValidDates.EarliestClinicalDate) };

```

Comment:
So very strangely, there was only an issue with localization in here, even though when called directly from a method it's fine. I think it's because static files are loaded earlier than the startup method is run. Not really sure exactly how to fix this (i.e. where else to specify the localization), so i decided to just split the test theory into two separate facts to avoid spending ages on it...

Reply:
Hmm, this would suggest that `dotnet` is setting the locale based on the request, which isn't too hard to believe...
Perhaps the conclusion is that the `DateValidityRange` should not directly depend on the `DateTime.Now`..?
But yeah, the split into two Facts seems very prudent for this.